changelog/@unreleased/pr-4229.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: `CqlClient` interface and its implementation along with `CqlClientManager` interface.
```

Comment:
Not particularly fussed, but I feel like until this feature is released as a whole, probably avoid change log. Thoughts @jeremyk-91